Makefile

```diff
@@ -1,28 +1,25 @@
-all: vendor
+all:
 	go build -o cni-noop ./cmd/cni-noop
 	go build -o cnispawn ./cmd/cnispawn
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
-glide:
-	@which glide || go get -u github.com/Masterminds/glide
-
+vendor: glide.lock | glide
+	glide --quiet install --strip-vendor
 glide.lock: glide.yaml | glide
```

Comment:
I am not sure about this dependency on `glide.yml` from `glide.lock`. When git-cloning the kubeadm-nspawn repository for the first time, the vendor/ directory is not yet populated. The user has to run `glide install`. But running `make vendor` will not work because at the moment `glide.lock` is older than `glide.yaml` in the repo:

![image](https://cloud.githubusercontent.com/assets/27575/26313206/6962e3aa-3f0a-11e7-9e61-c41f04c7d62c.png)

So this will trigger the rule `glide update` and it will likely not work for new users who just cloned the repository.

Maybe using a [order-only-prerequisites](https://www.gnu.org/software/make/manual/make.html#Prerequisite-Types) like this (see the position of the pipe):
```
glide.lock: | glide.yaml glide
```

Untested.